src/ambient.d.ts

```diff
@@ -0,0 +1,22 @@
+declare global {
+  type FormResponse = {
```

Comment:
Name is too general, and this is really only used for AccountUpdateResult.

Let's use a type AccountUpdateResult or FormAccountUpdateResult, and make it optional where used (instead of nested optional accountUpdateResult)?

Reply:
Fixed (see comment)